Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -276,36 +226,72 @@ - (void)updateNavigationBar {
     if (bSearching) {
         // Add a Cancel button to the toolbar, only needed for iPads
         if ([UIDevice isPad]) {
-            if (!iPadCancelButton) {
-                iPadCancelButton = [[UIBarButtonItem alloc] initWithTitle:NSLocalizedString(@"Cancel", @"Verb - dismiss the notes search view")
-                                                 style:UIBarButtonItemStylePlain
-                                                target:self
-                                                action:@selector(cancelSearchButtonAction:)];
-            }
-            
-            [self.navigationItem setRightBarButtonItem:iPadCancelButton animated:YES];
+            [self.navigationItem setRightBarButtonItem:_iPadCancelButton animated:YES];
         } else {
            [self.navigationItem setRightBarButtonItem:nil animated:YES];
         }
         
         [self.navigationItem setLeftBarButtonItem:nil animated:YES];
     } else if (tagFilterType == SPTagFilterTypeDeleted) {
-        [self.navigationItem setRightBarButtonItem:emptyTrashButton animated:YES];
-        [self.navigationItem setLeftBarButtonItem:sidebarButton animated:YES];
+        [self.navigationItem setRightBarButtonItem:_emptyTrashButton animated:YES];
+        [self.navigationItem setLeftBarButtonItem:_sidebarButton animated:YES];
     } else {
-        [self.navigationItem setRightBarButtonItem:addButton animated:YES];
-        [self.navigationItem setLeftBarButtonItem:sidebarButton animated:YES];
+        [self.navigationItem setRightBarButtonItem:_addButton animated:YES];
+        [self.navigationItem setLeftBarButtonItem:_sidebarButton animated:YES];
     }
-    
+
     self.navigationItem.titleView = _searchBarContainer;
     self.navigationItem.titleView.hidden = NO;
-    
+
     // Title must be set to an empty string because we're using a custom titleView,
     // and otherwise we get odd navigation bar behaviour when pushing view controllers
     // onto the navigation controller after the notes editor.
     self.navigationItem.title = @"";
 }
 
+- (void)configureNavigationButtons {
+
+    /// Button: New Note
+    ///
+    self.addButton = [UIBarButtonItem barButtonWithImage:[UIImage imageNamed:@"icon_new_note"]
+                                          imageAlignment:UIBarButtonImageAlignmentRight
+                                                  target:self
+                                                selector:@selector(addButtonAction:)];
+    _addButton.isAccessibilityElement = YES;
```

Comment:
Why use the iVar instead of the property?

Reply:
Mostly because... it's being set in the instruction immediately before, and this way you avoid going thru the getter.

**BUT** that being said, for the sake of consistency!! Updating all of the ivar indirections to access directly the properties.

Thank you for bringing this up!